go.mod

```diff
@@ -0,0 +1,45 @@
+module github.com/zhanggbj/kperf
+
+go 1.14
+
+require (
+	github.com/kr/text v0.2.0 // indirect
+	github.com/mitchellh/go-homedir v1.1.0
+	github.com/montanaflynn/stats v0.6.3
+	github.com/niemeyer/pretty v0.0.0-20200227124842-a10e7caefd8e // indirect
+	github.com/pelletier/go-toml v1.8.0 // indirect
+	github.com/spf13/cobra v0.0.6
+	github.com/spf13/viper v1.6.2
+	gopkg.in/check.v1 v1.0.0-20200227125254-8fa46927fb4f // indirect
+	k8s.io/api v0.18.1
+	k8s.io/apimachinery v0.18.1
+	k8s.io/client-go v12.0.0+incompatible
+	k8s.io/kubernetes v1.15.4
+	knative.dev/networking v0.0.0-20200703165942-2f7b2a7eca36
+	knative.dev/pkg v0.0.0-20200702222342-ea4d6e985ba0
+	knative.dev/serving v0.15.2
+)
+
+// Pinned to kubernetes-1.15.4
+replace (
+	k8s.io/api => k8s.io/api v0.0.0-20190918195907-bd6ac527cfd2
```

Comment:
Better to pin to the same k8s version as other Knative repos, currently serving is pinning to v0.18.8 - https://github.com/knative/serving/blob/master/go.mod#L44

Reply:
I haven't tested this yet, how about I open a separate task after this initial PR is merged and update it?
https://github.com/knative-sandbox/kperf/issues/6